ansible/roles/packages/tasks/install-redhat.yaml

```diff
@@ -1,8 +1,4 @@
 ---
-
```

Comment:
instead updating cache while installing the RPM. That will update cache for only repository that are explicitly enabled while installing RPMs.